.github/PULL_REQUEST-TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+# Pull Request for Hacktobot Project 
+## Issue Title : <!-- Enter the issue title here -->
+
+- **Name:** <!--Mention Your name-->
+- **GitHub ID:** <!-- Mention your GitHub ID -->
+- **Idenitfy yourself:** <!-- Mention your role -->
```

Comment:
I think there isn't any need to mention the role. Please remove that. Besides that, it looks good.